packages/airview-compliance-ui/src/features/application-tile/application-tile.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Box from "@mui/material/Box";
```

Comment:
For consistency, import as a named import: 

`import { Box } from "@mui/material";`